vic/vic_run/include/vic_run.h

```diff
@@ -144,13 +144,19 @@ double ErrorIcePackEnergyBalance(double Tsurf, ...);
 double ErrorPrintIcePackEnergyBalance(double, va_list);
 int ErrorPrintSnowPackEnergyBalance(double, va_list);
 int ErrorSnowPackEnergyBalance(double Tsurf, ...);
-int estimate_layer_ice_content(layer_data_struct *, double *, double *,
-                               double *, double *, double *, double *, double *,
-                               double, size_t, size_t, char);
-int estimate_layer_ice_content_quick_flux(layer_data_struct *, double *, double,
-                                          double, double, double, double *,
-                                          double *, double *, double *, double,
-                                          char);
+void estimate_frost_temperature_and_depth(double tmpT[MAX_LAYERS][MAX_NODES][MAX_FROST_AREAS + 1], double tmpZ[MAX_LAYERS][MAX_NODES], double *, double *, double *, double *, double, size_t,
```

Comment:
@bartnijssen I've tried to use pointer-type format to declare these variables but without success. The reason seems to be a multidimensional static array (e.g., `var[10][10]`) is not convertible to a multi-pointer (e.g., `double **`). I could change the structure of these variables and assign them as pointer to pointer instead of a static array - do you think it's necessary here?